### ISSUE TITLE
Realign Spring Function Center

### ISSUE THREAD

**Issue by Papermanzero** [AntiMicroX/antimicrox#648]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

The spring function for the mouse is currently always centered (middle of the screen).
it would be beneficial for some games to adjust the center to a different position.

**Comment by 6XGate** [comment]
I would second that. It seems that if the game is running in a resolution lower than the desktop, even if X or Wayland hasn't changed resolutions; the center is now off to the lower right. I'm having this issue with Wing Commander Kilrathi Saga. WC1 run at 320×200, but is scaled up to 1920×1080. The mouse is somehow scale in a way that it never appears on screen using spring.

**Comment by Papermanzero** [comment]
Yes, this feature is really required to align some games which have a different centre position
